app/src/ui/diff/index.tsx

```diff
@@ -952,6 +974,62 @@ export class Diff extends React.Component<IDiffProps, {}> {
     return null
   }
 
+  private renderLargeTextDiff() {
+    return (
+      <div className="panel empty large-diff">
+        <img src={NoDiffImage} />
+        <p>
+          The diff is too large to be displayed by default.
+          <br />
+          You can try to show it anyways, but performance may be negatively
+          impacted.
+        </p>
+        <Button onClick={this.showLargeDiff}>Show diff</Button>
+      </div>
+    )
+  }
+
+  private renderDiffTooLarge() {
+    return (
+      <div className="panel empty large-diff">
+        <img src={NoDiffImage} />
+        <p>The diff is too large to be displayed.</p>
```

Comment:
Could you help me understand why we removed the helper text: `The diff returned by Git is {diffSizeMB}MB ({diff.length} bytes), which is larger than what can be displayed in GitHub Desktop.` I thought it was a nice touch. Maybe even helping people be aware of what the limit is since it's a const.

Reply:
I asked @desktop/design for their input and they produced some awesome mocks for me to work from.

Reply:
I don't find the actual size of the diff helpful other than that it being a very large number. On top of that, it's information that our users don't necessarily have control over (e.g., they can't reduce the size of the diff itself, as far as I know.)

I'm more in favor of the terse "this is the situation right now. You can choose to bypass this, but this effect might happen as a result" type of messaging.